readthedocs/builds/version_slug.py

```diff
@@ -33,16 +33,16 @@
 VERSION_SLUG_REGEX = '(?:[a-z0-9][-._a-z0-9]+?)'
 
 
-version_slug_regex = re.compile(VERSION_SLUG_REGEX)
-
-
 class VersionSlugField(models.CharField):
     """
     Implementation inspired by ``django_extensions.db.fields.AutoSlugField``.
     """
 
-    allowed_chars = string.lowercase + string.digits + '-._'
+    allowed_punctuation = '-._'
+    allowed_chars = string.lowercase + string.digits + allowed_punctuation
     placeholder = '-'
+    fallback_slug = 'unkown'
```

Comment:
Doh! But the tests were green ... because I tested for the misspelled word :) Is fixed now.